Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorLevelTracingEffect.py

```diff
@@ -163,11 +163,9 @@ def preview(self,xy):
     ijk = self.effect.xyToIjk(xy, self.sliceWidget, masterImageData, parentTransformNode)
     dimensions = masterImageData.GetDimensions()
 
-    for index in range(3):
-      # TracingFilter crashes if it receives a seed point at the edge of the image,
-      # so only accept the point if it is inside the image and is at least one pixel away from the edge
-      if ijk[index] < 1 or ijk[index] >= dimensions[index]-1:
-        return
+    # Current LevelTracing implementation is 2D. In case of 3D implementation in future,
```

Comment:
I've tested this and without the check the filter makes the application crash (using CTChest sample data set, red slice view, when I move the mouse at the anterior boundary of the image near the). However, it does not seem to be hard to add a check in the filter to prevent the crash. I'll have a look.

Reply:
It is weird, I tried on many cases and it didn't crash. Thanks for looking deep into the check and having this merged